docs/troubleshooting.md

```diff
@@ -42,3 +42,23 @@ to be included as part of the test.
 
 [use-unicorn-pr]: https://github.com/alphagov/router-api/pull/113
 [clear-page-cache-pr]: https://github.com/alphagov/publishing-e2e-tests/pull/204
+
+### Docker disk space
+
+Docker limits the amount of disk space it uses. This sometimes results in
+rather opaque errors when you try and run tasks - generally related to
+errors installing Mongo or Postres. One example is
+`Moped::Errors::ConnectionFailure: Could not connect to a primary node for
+replica set`.
+
+The most reliable way to fix this is to find and remove unnecessary Docker
+containers and images.
+
+```
+docker rmi $(docker images -f dangling=true -q)
+docker volume rm $(docker volume ls -q -f dangling=true)
+```
+
+Docker for Mac will start comsuming vast amounts of CPU when it isn't given
+enough RAM.   If you find the apps aren't booting within the 60 second timeout
```

Comment:
```suggestion
enough RAM. If you find the apps aren't booting within the 60 second timeout
```

Reply:
Fixed in https://github.com/alphagov/publishing-e2e-tests/pull/467/commits/f669e11bc9071792a64981e6ed12047f4d765057.